gulp/browser-test.js

```diff
@@ -40,6 +40,6 @@ function browserWatch() {
   gulp.watch(['src/**/*.js', 'test/**/*.js'], ['browser-bundle']);
 }
 
-gulp.task('browser-bundle', ['lint-src', 'lint-test'],   bundle);
+gulp.task('browser-bundle', [],   bundle);
```

Comment:
don't like the linting?  We could probably make these run async to the tests, but their sync because often a linting error can break the build.